engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -432,6 +443,7 @@ static void LogFrameBufferError(GLenum status)
             GL_FLOAT_MAT4,
             GL_SAMPLER_2D,
             GL_SAMPLER_CUBE,
+            DMGRAPHICS_SAMPLER_2D_ARRAY,
```

Comment:
The OpenGL enum for the array sampler doesn't exist in es2, but we can still use it